### ISSUE TITLE
passport representatives

### ISSUE THREAD

**Issue by cscmnu** [michaelmusty/BelyiDB#13]
The passport representatives s_1,s_2,s_3 of https://beta.lmfdb.org/Belyi/6T3/%5B6%2C2%2C2%5D/6/222/2211/g0/a
do not satisfy s_3*s_2*s_1=1. We have s_2*s_3*s_1 =1 but the function BelyiMap([s_1,s_3,s_2]) produces the error that it has failed a sanity check, but (s_2*s_3*s_1)^-1 = 1 and BelyiMap([s_1^-1, s_2^-1, s_3^-1]) works fine.

**Comment by jvoight** [comment]
Please help:

> sigma := [Sym(6) | (1,2,3,4,5,6), (1,4)(2,3)(5,6), (1,5)(2,4)]; 
> sigma[3]*sigma[2]*sigma[1];
Id($)

Seems OK to me.

**Comment by jvoight** [comment]
There is an issue with a different triple in the passport.  

> sigma := [Sym(6) | (1, 2)(4, 5),
> (1, 4)(2, 6)(3, 5),
> (1, 6, 2, 4, 3, 5)
> ]; 
> sigma[3]*sigma[2]*sigma[1];
Id($)
> BelyiMap(sigma);
Chebyshev spherical Belyi map
Local Sanity Check Failed for p=9721:
sigma = 
[
(1, 2)(4, 5),
(1, 4)(2, 6)(3, 5),
(1, 6, 2, 4, 3, 5)
].
supp(phi) = 
[
Place at (0 : 1),
Place at (7086 : 1),
Place at (2635 : 1),
Place at (9720 : 1),
Place at (4861 : 1),
Place at (4860 : 1),
Place at (1 : 1)
]
[ 2, 2, 2, -1, -2, -2, -1 ].
supp(phi-1) = 
[
Place at (1 : 0),
Place at (9720 : 1),
Place at (4861 : 1),
Place at (4860 : 1),
Place at (1 : 1)
]
[ 6, -1, -2, -2, -1 ].
Curve over Rational Field defined by
0
(16*x^6 - 24*x^4 + 9*x^2)/(16*x^6 - 24*x^4 + 9*x^2 - 1)
Runtime error: Failed sanity check, so the Belyi map is wrong.  This shouldn't 
happen.

The sanity check fails but not because the map is wrong: it looks like it wants the ramification at a different place, so the usual S_3 issue.

**Comment by michaelmusty** [comment]
Yeah it looks like I implemented the local version of the sanity check
without a lax (up to S_3 action) option. I see 3 options:

1. redo the hardcoded spherical map in nonhyperbolic.m to match the triple.
2. add ability to test lax isomorphism in sanity_check_local.m
3. for spherical maps it should be OK to use "regular" (non-local) sanity
check which has optional lax := true argument.

Mike

**Comment by cscmnu** [comment]
Yes sorry I meant a different triple in the passport. The examples I came across where sigma[3]*sigma[2]*sigma[1] is not the identity but BelyiMap() works are:

s1:=[ Sym(3) |
    (1, 3, 2),
    (2, 3),
    (1, 2)
];
> &*Reverse(s1);
(1, 2, 3)
> BelyiMap(s1); 
Curve over Rational Field defined by
0
2/(4*x^3 - 3*x + 1)
s2:=[ Sym(6) |
    (1, 3, 2)(4, 6, 5),
    (1, 4)(2, 6)(3, 5),
    (1, 5)(2, 4)(3, 6)
];
> &*Reverse(s2);
(1, 2, 3)(4, 5, 6)
> BelyiMap(s2);
Curve over Rational Field defined by
0
-4*x^3/(x^6 - 2*x^3 + 1)
s3:=[ Sym(6) | 
    (1, 6, 2, 4, 3, 5),
    (1, 4)(2, 6)(3, 5),
    (1, 2)(4, 5)
];
> &*Reverse(s3);
(1, 2, 3)(4, 5, 6)
> BelyiMap(s3);
Curve over Rational Field defined by
0
-1/(16*x^6 - 24*x^4 + 9*x^2 - 1)
s4:=[ Sym(12) |
    (1, 9, 2, 7, 3, 8)(4, 12, 5, 10, 6, 11),
    (1, 4)(2, 6)(3, 5)(7, 10)(8, 12)(9, 11),
    (1, 11)(2, 10)(3, 12)(4, 8)(5, 7)(6, 9)
];
> &*Reverse(s4);
(1, 2, 3)(4, 5, 6)(7, 8, 9)(10, 11, 12)
> BelyiMap(s4);
Curve over Rational Field defined by
0
-4*x^6/(x^12 - 2*x^6 + 1)
